### ISSUE TITLE
No indication of sent game request

### ISSUE THREAD

**Issue by Zomis** [Cardshifter/HTML-Client#86]
When sending an invitation to someone, there's no indication that you have done so. Also related to https://github.com/Cardshifter/Cardshifter/issues/373

**Comment by Phrancis** [comment]
@Zomis does the game server broadcast a message to all connected clients when a game invite is sent between 2 players?

**Comment by Zomis** [comment]
No, only the players that are involved in the invitation is notified.
